careers/university/templates/university/index.html

```diff
@@ -233,8 +233,12 @@ <h2 class="ally">Apply</h2>
     </div>
   {% else %}
     <div class="contain apply-closed">
-      <h3 class="apply">We don't have any internship positions open right now.</h3>
-      <p>But check back with us soon! We typically accept applications from September through April. If you'd like to get a head start on applying, feel free to <a href="mailto:university@mozilla.com">email us</a> your resume and tell us why you are interested in an internship with Mozilla.</p>
+      <h3 class="apply">We've finished our internship hiring for this summer.</h3>
+      <p>
+        But check back with us soon! We typically accept applications from September through March.
+        If you're still interested in contributing to the Project, we'd love for you to get
+        <a href=" https://www.mozilla.org/en-US/contribute/">involved</a>.
```

Comment:
Should probably remove the <code>/en-US</code> from this URL so it's locale-agnostic. I'd also suggesting putting "get" inside the link.